pandas/_libs/tslibs/conversion.pyx

```diff
@@ -203,9 +203,13 @@ cdef class _TSObject:
     #    int64_t value               # numpy dt64
     #    object tzinfo
 
-    property value:
-        def __get__(self):
-            return self.value
+    @property
+    def value(self):
+        return self.value
+
+    @value.setter
```

Comment:
don’t add this

Reply:
Yeah, some 2.7 builds failed earlier without it, and it looked like there were places where `value` was being set, so I added that to see if it would resolve the issue since I couldn't reproduce locally.  Looks like that wasn't the issue.